### DISCUSSION TITLE
Scalene in Kubernetes

### DISCUSSION THREAD

**Discussion by ducminhle** [plasma-umass/scalene#716] [Q&A]
Hi Team,
Do we have a solution to use Scalene inside a container running in Kubernetes?  Or integrate with some Continuous Profiling solution (Ex: [parca](https://github.com/parca-dev/parca), [pyroscope](https://github.com/grafana/pyroscope), ...)?
I have some problems with memory and GPU in Kubernetes, but I can not reproduce the error on the local machine ... I have read scalene document, but can not find the document for the deployment application inside container in Kubernetes.

**Comment by raayu83** [comment]
I'm also trying to run scalene inside Kubernetes.
All Columns except Time Python and system are empty.
Is tracking memory consumption not possible in Kuberenetes?
